### PR TITLE
add floddfill in js

### PR DESCRIPTION
**Pull request by AmanParmar-git**
## Have you read the Contribution Guidelines?

Yes

## Description

Flood Fill implementation in javascript. basic idea is first you give it starting pixel then it will change pixels 4-directionally until it can.

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #647

### CODE REVIEW

**Review by mananchawla2005 — CHANGES_REQUESTED** [review]
Conflicts in readme

**Comment by AmanParmar-git** [comment]
Maintainers , respectively check if readme is in alphabetical order don't merge pr directly.

**Pull request merged**
